Extensions/AdMob/admobtools.ts

```diff
@@ -333,6 +333,7 @@ namespace gdjs {
         adUnitId,
         position: atTop ? 'top' : 'bottom',
         size: bannerRequestedAdSizeType,
+        offset: 0,
```

Comment:
You are not using the parameter you added in the action declaration. It should be added in the `setupBanner` signature

Reply:
As discussed on the other PR, this is valid but the parameter can be removed entirely ;)